server/integration/testsuite/src/test/java/org/infinispan/server/test/jcache/JCacheStateAwarenessIT.java

```diff
@@ -0,0 +1,212 @@
+   private static final String enableManagementTestCacheName = "enableManagementTestCache";
+   private static final String getCacheNamesTestCacheName = "getCacheNamesTestCache";
+   private static final String getCacheTestCacheName = "getCacheTestCache";
+
+   private static final String testKey = "testKey";
+   private static final String testValue = "testValue";
+
+   final int managementPort = 9999;
+
+   @InfinispanResource("cachecontainer")
+   private RemoteInfinispanServer server1;
+   private RemoteCacheManager rcm1;
+
+   private CachingProvider jcacheProvider;
+   private CacheManager cacheManager;
+   private MBeanServer mBeanServer;
+
+   @Before
+   public void setUp() {
+      if (rcm1 == null) {
+         Configuration conf = new ConfigurationBuilder().addServer()
+               .host(server1.getHotrodEndpoint().getInetAddress().getHostName())
+               .port(server1.getHotrodEndpoint().getPort()).build();
+
+         rcm1 = new RemoteCacheManager(conf);
+
+         jcacheProvider = Caching.getCachingProvider("org.infinispan.jcache.remote.JCachingProvider");
+         cacheManager = jcacheProvider.getCacheManager();
+
+         mBeanServer = ManagementFactory.getPlatformMBeanServer();
+
+      }
+   }
+
+   /**
+    * Test enableStatistics command on JCache API <br>
+    * <br>
+    * Fix pending in <b>ISPN-7095</b>, needs to be uncommented when implemented
+    * 
+    * @throws ReflectionException
+    * @throws MalformedObjectNameException
+    * @throws InstanceNotFoundException
+    * @throws IntrospectionException
+    * @throws MBeanException
+    * @throws AttributeNotFoundException
+    */
+   // @Test
+   public void testEnableStatistics() throws IntrospectionException, InstanceNotFoundException,
+         MalformedObjectNameException, ReflectionException, AttributeNotFoundException, MBeanException {
+      String mBeanName = getMBeanName(enableStatisticsTestCacheName);
+
+      try {
+         cacheManager.enableStatistics(enableStatisticsTestCacheName, false);
+      } catch (NullPointerException npe) {
+         npe.printStackTrace();
+         fail("NullPointerException thrown while attempting to disable statistics");
+      }
+
+      assertTrue("Statistics register as enabled after disabling",
+            !(Boolean) mBeanServer.getAttribute(new ObjectName(mBeanName), "StatisticsEnabled"));
+
+      try {
+         cacheManager.enableStatistics(enableStatisticsTestCacheName, true);
+      } catch (NullPointerException npe) {
+         npe.printStackTrace();
+         fail("NullPointerException thrown while attempting to enable statistics");
+      }
+
+      assertTrue("Statistics register as disabled after enabling",
+            (Boolean) mBeanServer.getAttribute(new ObjectName(mBeanName), "StatisticsEnabled"));
+
+   }
+
+   /**
+    * Test enableManagement command on JCache API<br>
+    * <br>
+    * Fix pending in <b>ISPN-7095</b>, needs to be uncommented when implemented
+    * 
+    */
+   // @Test
```

Comment:
There will get removed when ISPN-7095 is finished, seems smarter to leave them in for clarity